packages/wrangler/src/api/startDevWorker/utils.ts

```diff
@@ -308,8 +309,22 @@ export function convertCfWorkerInitBindingsToBindings(
 	return output;
 }
 
+/**
+ * Convert either StartDevWorkerOptions["bindings"] or WorkerMetadataBinding[] to CfWorkerInit["bindings"]
```

Comment:
nit: I like when comments start with a one sentence high level summary of what the function does